crates/oxc_linter/src/snapshots/no_useless_escape.snap

```diff
@@ -14,160 +14,160 @@ expression: no_useless_escape
    ·            ──
    ╰────
 
-  ⚠ eslint(no-useless-escape): Unnecessary escape character '\''
+  ⚠ eslint(no-useless-escape): Unnecessary escape character '\\'
    ╭─[no_useless_escape.tsx:1:1]
  1 │ var foo = "\'";
-   ·            ──
+   ·           ──
    ╰────
 
-  ⚠ eslint(no-useless-escape): Unnecessary escape character '#'
+  ⚠ eslint(no-useless-escape): Unnecessary escape character '\\'
    ╭─[no_useless_escape.tsx:1:1]
  1 │ var foo = "\#/";
-   ·            ──
+   ·           ──
```

Comment:
It seems like there is a off-by-1 error here, it should point at `\#` and the message should say `#`